tests/unit/models/test_model_interfaces.py

```diff
@@ -855,5 +834,9 @@ def test_squeeze_hyperparameters_when_param_at_edge_of_bounds() -> None:
     kernel.lengthscales = gpflow.Parameter(
         kernel.lengthscales, transform=tfp.bijectors.Sigmoid(low=lower, high=upper)
     )
-    squeeze_hyperparameters(kernel, 0.1)
+    squeeze_hyperparameters(kernel, alpha=0.1)
     npt.assert_array_almost_equal(kernel.lengthscales, [0.1 + 4e-2, 0.5 - 4e-2])
+
+    lik = gpflow.likelihoods.Gaussian(variance=1.01e-6)
```

Comment:
I would put this in a separate test. Something like `test_squeeze_hyperparameters_softplus` and rename the other test something like `test_squeeze_hyperparameters_sigmoid`.

Also would be good to add a simple test that confirms that passing in a 0 or negative epsilon results in a ValueError.